apps/mark-scan/backend/src/custom-paper-handler/test_utils.ts

```diff
@@ -9,7 +9,7 @@ import { Buffer } from 'buffer';
 const BALLOT_FIXTURE_FRONT_PATH = join(
   __dirname,
   'fixtures',
-  'ballot-pdf-data.bin'
```

Comment:
Same ballot with a large top margin to more realistically represent markscan ballots. `ballot-pdf-data.bin` is now only used by the state machine CLI which is unmaintained and should be removed in a separate PR.